site/data/refDataApi.ts

```diff
@@ -45,6 +45,7 @@ export const fetchStops = async (input: FetchStopsInput) => {
 interface FetchServicesInput {
     adminAreaCodes?: string[];
     dataSource?: Modes;
+    modes?: string[];
```

Comment:
Then we can make this an interface called Mode with the transport values

Reply:
done